recipes/expertai-extract/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "expertai-extract" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/expertai-extract-{{ version }}.tar.gz
+  sha256: 32043e2fc377fd04b56b6086f7d9ce4c5c13f975d5a520af01eaf8b519593ad5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - python-dateutil
+    - urllib3 >=1.25.3
```

Comment:
Double check that requirements are complete. Theres a requirments.txt in the repo that lists more:

```
python_dateutil >= 2.5.3
setuptools >= 21.0.0
urllib3 >= 1.25.3

pytest >= 7.1.1
requests >= 2.27.1
```

Reply:
Daniel, we had updated the requirements section and also renamed (removed) requirements.txt to requirements-dev.txt, since we need them only for development purposes.